gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixUtils.java

```diff
@@ -145,6 +146,31 @@ static void waitJobInitialization(
     log.info("Work flow {} initialized", workFlowName);
   }
 
+  public static boolean deleteTaskFromHelixJob(String workFlowName,
+      String jobName, String taskID, TaskDriver helixTaskDriver) {
+    try {
+      log.info(String.format("try to delete task %s from workflow %s, job %s", taskID, workFlowName, jobName));
+      helixTaskDriver.deleteTask(workFlowName, jobName, taskID);
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
why handle in this way ? why not propagated to caller and let them determine since this is just a util method ?

Reply:
Just want to wrap all the delete / add logic in the util method, and as the doc mentioned, if there is exception it's caller's responsibility to determine whether it succeed.